distributed/scheduler.py

```diff
@@ -410,16 +410,19 @@ def get_versions(self, comm):
         """ Basic information about ourselves and our cluster """
         return get_versions()
 
-    def start_services(self, listen_ip=''):
+    def start_services(self, host):
         for k, v in self.service_specs.items():
             if isinstance(k, tuple):
                 k, port = k
             else:
                 port = 0
 
+            if host == '0.0.0.0':
+                host = ''  # for IPv6
```

Comment:
This failed otherwise

```python
def test_defaults(loop):
    with popen(['dask-scheduler', '--no-bokeh']) as proc:

        @gen.coroutine
        def f():
            # Default behaviour is to listen on all addresses
            yield [
                assert_can_connect_from_everywhere_4_6(8786, 2.0),  # main port
                assert_can_connect_from_everywhere_4_6(9786, 2.0),  # HTTP port <<<<----------
                ]

        loop.run_sync(f)
```